File: CodingTestPlatform/BOJ/1_Bronze/2/15552/15552.py
# 빠른 A+B
# https://www.acmicpc.net/problem/15552

import sys
input = sys.stdin.readline

cnt = int(input().rstrip())

for i in range(cnt):
    num1, num2 = map(int, input().split())
    print(num1 + num2)

File: CodingTestPlatform/BOJ/1_Bronze/2/15552/15552.js
/**
 * 빠른 A+B
 * Bonze2
 * https://www.acmicpc.net/problem/15552
 */

const [T, ...num] = require('fs').readFileSync('/dev/stdin').toString().trim().split("\n");

const result = num.map((str) => {
    const num = str.split(" ").map((char) => parseInt(char));
    return num[0] + num[1];
});

console.log(result.join("\n"));

